config/i18n/all-langs.ts

```diff
@@ -148,7 +148,7 @@ export const i18nextCodes = {
 };
 
 // These are for the language selector dropdown menu in the footer
-export const langDisplayNames = {
+export const langDisplayNames: Record<string, string> = {
```

Comment:
```suggestion
export const langDisplayNames = {
```
This change would reduce type safety.  There's a few ways to go here, but the DRYest approach I can think of is to use an enum like this

```ts
export enum LangNames {
  English = 'english',
  Espanol = 'espanol',
  ...
}
```
And then wherever we have the string `'english'` we'd have `LangNames.English`.  This should cut down on the number of hoops we need to jump through to convince TS we know what we're doing.